inc/front/cdn.php

```diff
@@ -269,7 +269,7 @@ function rocket_cdn_custom_files( $html ) {
 		 *
 		 * @param array $filetypes Array of file types.
 		 */
-		$filetypes = apply_filters( 'rocket_cdn_custom_filetypes', array( 'mp3', 'ogg', 'mp4', 'm4v', 'avi', 'mov', 'flv', 'swf', 'webm', 'pdf', 'doc', 'docx', 'txt', 'zip', 'tar', 'bz2', 'tgz', 'rar' ) );
+		$filetypes = apply_filters( 'rocket_cdn_custom_filetypes', array( 'mp3', 'ogg', 'mp4', 'm4v', 'avi', 'mov', 'flv', 'swf', 'webm', 'pdf', 'doc', 'docx', 'txt', 'zip', 'tar', 'bz2', 'tgz', 'rar', 'jpg', 'jpeg', 'png', 'gif', 'webp', 'bmp', 'tiff' ) );
```

Comment:
I'm trying to cover common filetypes with the default values, if customers need more the filter is there for that. I don't think ico is really needed since we're matching `<a href` only here.